src/charts/stacked-bar.js

```diff
@@ -438,12 +433,11 @@ define(function(require){
             // Enter + Update
             let bars = barJoin
                     .enter()
-                      .append('rect')
-                        .classed('bar', true)
-                        .attr('x', (d) => xScale(d[0]) )
-                        .attr('y', (d) => yScale(d.data.key) )
-                        .attr('height', yScale.bandwidth())
-                        .attr('fill', (({data}) => categoryColorMap[`${data.stack}${data.key}`]));
+                    .append('rect')
```

Comment:
The previous indentation was actually on purpose, check https://github.com/eventbrite/britecharts/blob/master/CODESTYLEGUIDE.md#append-indentation for a rationale